src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const value in robot) {
+    const keys = robot[value];
```

Comment:
This confused me a little. This is a single value, but from its name it seems it's in plural.
```suggestion
    const key = robot[value];
```